sycl/test-e2e/NewOffloadDriver/Inputs/split-per-source-second-file.cpp

```diff
@@ -0,0 +1,23 @@
+#include "split-per-source.h"
```

Comment:
do we need a copyright header?

Reply:
This file was lifted from here: ./sycl/test-e2e/DeviceCodeSplit/Inputs/split-per-source-second-file.cpp where I do not find a copyright header.